source/includes/usage-examples/sample-write-app.c

```diff
@@ -0,0 +1,29 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int
+main (int argc, char *argv[])
```

Comment:
S: based on feedback I got on my PRs: since you don't use the argc and argv[] parameters, you can just pass void
```suggestion
main (void)
```

Reply:
Good to know! Thanks 🙏